dspace-api/src/main/java/cz/cuni/mff/ufal/health/CuratorCheck.java

```diff
@@ -95,7 +95,7 @@ private static String output_checklinks(ArrayList<String> results) {
                 if (str.trim().endsWith("- OK")) {
                     continue;
                 } else if (str.trim().startsWith("Item:")) {
-                    last_item = str;
+                    last_item = str.substring(0, str.indexOf(']')+1);
```

Comment:
@cyplas Do you have some before and after outputs for this change? I don't see what it does (ie. what's in str and why do you need to take substring to the first ']'). Thanks

Reply:
@kosarko Yes, for example:

```
BEFORE:
Item: 11356/1026  [https://beta.clarin.si/repository/xmlui/admin/item?itemID=1536] has 4 urls to check...
- http://creativecommons.org/licenses/by-sa/4.0/ = 403 - FAILED

AFTER:
Item: 11356/1026  [https://beta.clarin.si/repository/xmlui/admin/item?itemID=1536]
- http://creativecommons.org/licenses/by-sa/4.0/ = 403 - FAILED
```

Basically, I commented out the "has X urls to check". I think that the idea should be either to report on the number of URLs for all the handles (including the ones with no failures) or (as I've opted for) just list the failed URLs; showing the number of URLs but only for the handles with failures is confusing IMO until you go look at the healthcheck code.

Reply:
ok; thanks